src/apps/weblib/js-api/vizzu.js

```diff
@@ -73,7 +73,8 @@ export default class Vizzu {
       } catch (e) {
         if (Number.isInteger(e)) {
           let address = parseInt(e, 10);
-          let cMessage = this.module._vizzu_errorMessage(address);
+          let type = new this.module["ExceptionInfo"](address).get_type();
```

Comment:
no need for slower string lookup. this.module.ExceptionInfo should be suffice.

Reply:
True, thx